### DISCUSSION TITLE
Feature Request: Share Tags

### DISCUSSION THREAD

**Discussion by fitztheloyal** [standardnotes/forum#2503] [Suggestions]
I actually made this suggestion to Evernote, but they haven't implemented it yet. I think it's even more important for Standard Notes since you don't have "notebooks". Actually, I believe that this functionality would actually make the notebooks feature unnecessary and would leapfrog Evernote in this one issue. 

Create the ability to share a tag, thus everything tagged with that tag would be shared to a particular person or group. So in other words, if I create a tag that is called "Bookkeeper", and then I share this tag with Shelly (my bookkeeper), she would see all notes that I tag this way. I could unshare the tag at any time, and I could change who it is shared with. You could also add other shared tags to the same note. So if I had a tag named "Project Eagle" that I share with George and Annie, and then I tag "Note A" with both "Bookkeeper" and "Project Eagle", that note would be shared with Shelly, George and Annie.

**Comment by atmoio** [comment]
Sharing/collaboration is a huge undertaking and not currently something we're ready to take on. The closest thing is using the Listed extension to share an individual note. Perhaps we can extend Listed to allow publishing the contents of an entire tag. That's a cool idea. That might happen. But collaboration, that's years and years away, if at all. It's just such a difficult thing to get right.
